server/api/rounds.py

```diff
@@ -291,30 +291,29 @@ def sampled_ballot_interpretations_to_cvrs(contest: Contest) -> supersimple.CVRS
 def calculate_risk_measurements(election: Election, round: Round):
     if not election.risk_limit:  # Shouldn't happen, we need this for typechecking
         raise Exception("Risk limit not defined")  # pragma: no cover
```

Comment:
since this shouldn't happen, maybe switch to `assert election.risk_limit is not None`. im realizing that's a better pattern than having a custom exception